package/utils/busybox/Makefile

```
@@ -77,13 +77,24 @@ define Package/busybox/config
 endef
 
 ifdef CONFIG_BUSYBOX_CONFIG_FEATURE_SYSLOG
```

```suggestion
ifneq ($(CONFIG_BUSYBOX_$(BUSYBOX_SYM)_FEATURE_SYSLOG),)
```

I'm wondering if this has been broken for a while?